plugins/pulp_rpm/plugins/migrations/0011_new_importer.py

```diff
@@ -16,7 +15,7 @@
 
 
 def migrate(*args, **kwargs):
-    for type_id in (RPM.TYPE, SRPM.TYPE):
+    for type_id in ('rpm', 'srpm'):
```

Comment:
Since these are in an old migration we would do better to leave them hardcoded as strings. If we in the future change the value of `RPM._content_type_id.default` or `SRPM._content_type_id.default` it will have an unexpected side-effect on migration 11 causing it to fail.